### PR TITLE
Unit tests & bug fixes

### PR DESCRIPTION
**Pull request by stuckshut**
@mattlorimor

### CODE REVIEW

File: eris.py [hunk 1]
```diff
@@ -77,8 +76,10 @@ def is_direct_message(message):
     Returns:
         True if the message was a direct message. False if it was not.
     """
-    # If message.author is of type User, this was a direct message.
-    return type(message.author) == discord.User
+    # If message.channel is a PrivateChannel, then it's a direct message
```
Comment (mattlorimor) [inline eris.py]: Nice!

File: tests/test_commands.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
+from unittest import TestCase
+from unittest.mock import MagicMock
```
Comment (mattlorimor) [inline tests/test_commands.py]: This should just be ```from mock import MagicMock```.

File: tests/test_eris.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+from unittest import TestCase
+from unittest.mock import patch, MagicMock
```
Comment (mattlorimor) [inline tests/test_eris.py]: ```from mock import patch, MagicMock```

File: tests/test_commands.py (hunk 2 above)
Comment (stuckshut) [inline tests/test_commands.py]: mock was actually added to the standard library in 3.3 as unittest.mock

**Comment by mattlorimor** [comment]
Looks good. Tests pass locally.

+1

**Pull request merged**